src/NServiceBus.Transport.RabbitMQ/RabbitMQClusterTransport.cs

```diff
@@ -15,5 +15,14 @@ public RabbitMQClusterTransport(Topology topology, string connectionString, Queu
             : base(GetBuiltInTopology(topology), connectionString, queueMode)
         {
         }
+
+        /// <summary>
+        /// Adds a new cluster node
+        /// </summary>
+        /// <param name="hostName">An additional hostname of a node in the cluster.</param>
+        public void AddNode(string hostName)
```

Comment:
so there is no option to change the port of the additional hosts?

Reply:
Correct. The `CreateConnection` method accepts either a list of hostnames, or a list of `AmqpTcpEndpoint`s.

Theoretically, using `AmqpTcpEndpoint` might allow different ports etc., but I'm not sure if that's even true. We set the port etc. on the `ConnectionFactory` class, so I'm not sure how we would set multiple of them.

We could add an overload to accept instances of `AmqpTcpEndpoint`s? It would mean leaking a rabbit class to our API though.